templates/definition/vehicle/bmw.yaml

```
@@ -4,9 +4,9 @@ products:
 requirements:
   description:
     de: |
-      Benötigt `hcaptcha` Token. Dieses muss einmalig unter https://bimmer-connected.readthedocs.io/en/latest/captcha/rest_of_world.html generiert werden. Das Token ist nur für kurze Zeit gültig. Bitte möglichst schnell nach Generierung in die Konfiguration kopieren und evcc starten.
+      Benötigt `hcaptcha` Token. Dieses muss einmalig unter [https://bimmer-connected.readthedocs.io/en/latest/captcha/rest&#95;of&#95;world.html](https://bimmer-connected.readthedocs.io/en/latest/captcha/rest_of_world.html) generiert werden. Das Token ist nur für kurze Zeit gültig. Bitte möglichst schnell nach Generierung in die Konfiguration kopieren und evcc starten.
```

Gerade langen Links sollten wir eher nen lesbaren Namen geben. Das machen wir auch an anderen Stellen noch nicht konsequent, ist aber ne gute Idee.

```suggestion
      Benötigt `hcaptcha` Token. Dieses muss einmalig unter [bimmer-connected](https://bimmer-connected.readthedocs.io/en/latest/captcha/rest_of_world.html) generiert werden. Das Token ist nur für kurze Zeit gültig. Bitte möglichst schnell nach Generierung in die Konfiguration kopieren und evcc starten.
```